.github/workflows/review-release-notes.yml

```diff
@@ -0,0 +1,37 @@
+name: Review Release Notes
+on: pull_request
+
+jobs:
+  release_notes_review:
+    runs-on: ubuntu-latest
+    if: github.repository == 'demisto/content' && github.event.pull_request.head.repo.fork == false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v2.0.0
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
```

Comment:
why not 3.10 ?

Reply:
Now that you're bringing it up, demisto-sdk is being tested against python 3.7 and 3.8, maybe it would be wise to install 3.8 here instead?

Reply:
yea lets go with 3.8